src/engine/Match_extract_mode.ml

```diff
@@ -255,6 +255,20 @@ let map_taint_trace map_loc traces =
       })
     traces
 
+let map_bindings map_loc bindings =
+  let map_tokens map_loc mval =
+    let mmval =
+      AST_generic_helpers.fix_token_locations_any map_loc
+        (Metavariable.mvalue_to_any mval)
+      |> Metavariable.mvalue_of_any
+    in
+    match mmval with
+    | Some m -> m
+    | None -> raise Common.Impossible
```

Comment:
I generally think `failwith "<msg>"` is better than Common.Impossible because at least you get some string, but I won't nitpick because we do have Common.Impossible in many places

Reply:
But i think its literally impossible right? it would mean that "to_any |> of_any" would have failed